### ISSUE TITLE
Visual and programmatic name properties are different for "Resources" combo box: A11y_Aspire Dashboard_Metrics_Label in Name..

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/aspire#10158]
### GitHubTags
#A11ySev2; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#BM_AspireDashboard_Web_June25;#Win32;#WCAG2.5.3;#WCAG2.1;#Label in Name;#A11yMAS;

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

### Repro Steps

1. Open Visual studio.
2. Open a aspire project and run it.
3. A new window will open in the chromium edge,
4. TAB to "Metrics" tab item and hit ENTER key.
5. Verify that Visual and programmatic name properties are different for "Resources" combo box.

### Actual Result:
Visual and programmatic name properties are different for "Resources" combo box

### Expected Result:
Name should be provided as "Resources".

### User Impact:
Speech input & text to speech users will face difficulty in accessing the control if visual label is different from programmatic name

### Attachment:

![Image](https://github.com/user-attachments/assets/7e7e5b2e-7d71-4767-8197-d11e5459cfeb)

**Comment by adamint** [comment]
The aria label for this combobox is actually _more_ descriptive than the visual label. This is on purpose, as though "Resources" is sufficient to describe the combobox given the visual context, it may not be as obvious to screen reader users.

This issue should be closed @AnKushSingh05
